### ISSUE TITLE
Build failed

### ISSUE THREAD

**Issue by tigercosmos** [szeged/blurdroid#5]
Follow the `README`, and set up the environment.  
```
tigercosmos@cantabile:~/blurdroid$ cargo build --target=arm-linux-androideabi
   Compiling blurdroid v0.1.4 (file:///home/u/tigercosmos/blurdroid)
error: failed to run custom build command for `blurdroid v0.1.4 (file:///home/u/tigercosmos/blurdroid)`
process didn't exit successfully: `/home/u/tigercosmos/blurdroid/target/debug/build/blurdroid-cd5a25ef561d80cc/build-script-build` (exit code: 101)
--- stderr
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error { repr: Os { code: 2, message: "No such file or directory" } }', /checkout/src/libcore/result.rs:906:4
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```

Also happened in servo/servo#19017

**Comment by zakorgy** [comment]
@tigercosmos the crate builds without problems on my machine. But I have an idea, what causes this error on your side.
First you should run with `RUST_BACKTRACE=1 cargo build --target=arm-linux-androideabi` and check if you have the following or similar backtrace: (If not, you should link the result you got)
```
error: failed to run custom build command for `blurdroid v0.1.4 (file:///home/zakorgy/work/blurdroid)`
process didn't exit successfully: `/home/zakorgy/work/blurdroid/target/debug/build/blurdroid-fc891dcf074a4e5c/build-script-build` (exit code: 101)
--- stderr
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error { repr: Os { code: 2, message: "No such file or directory" } }', /checkout/src/libcore/result.rs:906:4
stack backtrace:
   0: std::sys::imp::backtrace::tracing::imp::unwind_backtrace
             at /checkout/src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:49
   1: std::sys_common::backtrace::_print
             at /checkout/src/libstd/sys_common/backtrace.rs:69
   2: std::panicking::default_hook::{{closure}}
             at /checkout/src/libstd/sys_common/backtrace.rs:58
             at /checkout/src/libstd/panicking.rs:381
   3: std::panicking::default_hook
             at /checkout/src/libstd/panicking.rs:397
   4: std::panicking::rust_panic_with_hook
             at /checkout/src/libstd/panicking.rs:577
   5: std::panicking::begin_panic
             at /checkout/src/libstd/panicking.rs:538
   6: std::panicking::begin_panic_fmt
             at /checkout/src/libstd/panicking.rs:522
   7: rust_begin_unwind
             at /checkout/src/libstd/panicking.rs:498
   8: core::panicking::panic_fmt
             at /checkout/src/libcore/panicking.rs:71
   9: core::result::unwrap_failed
             at /checkout/src/libcore/macros.rs:23
  10: <core::result::Result<T, E>>::unwrap
             at /checkout/src/libcore/result.rs:772
  11: build_script_build::android_main
             at ./build.rs:35
  12: build_script_build::main
             at ./build.rs:10
  13: __rust_maybe_catch_panic
             at /checkout/src/libpanic_unwind/lib.rs:99
  14: std::rt::lang_start
             at /checkout/src/libstd/panicking.rs:459
             at /checkout/src/libstd/panic.rs:361
             at /checkout/src/libstd/rt.rs:59
  15: main
  16: __libc_start_main
  17: _start
```
If your backtrace also contains this part:
```  
11: build_script_build::android_main
             at ./build.rs:35
```
means that maybe your `ANDROID_NDK` variable points to the wrong path.
In short, you can search for your directory at `ANDROID_SDK` path, and check that it points to the directory, which contains the  `ndk-build` executable, because line 35 in build.rs try to use that.
FYI: I managed to produce the above error by changing the `ANDROID_NDK` path in my `.bashrc` from
`export ANDROID_NDK="/home/zakorgy/work/android/NDK/android-ndk-r12b"` to
`export ANDROID_NDK="/home/zakorgy/work/android/NDK`

Please contact me here, if you have any questions.

**Comment by tigercosmos** [comment]
It work with adding a `/` in the end of `ANDROID_NDK` path. It's weird.
Besides, then I got
```
tigercosmos@ubuntu:~/blurdroid$ cargo build --target=arm-linux-androideabi
   Compiling blurdroid v0.1.4 (file:///home/tigercosmos/blurdroid)
error[E0463]: can't find crate for `std`
  |
  = note: the `arm-linux-androideabi` target may not be installed

error: aborting due to previous error
```
I found that's caused by not installing target. Finally, it works :)
Shall we add notice about cross-compilation, such as using [rustup](https://github.com/rust-lang-nursery/rustup.rs#cross-compilation)?

**Comment by zakorgy** [comment]
I have updated the `README.md` according to that.

**Comment by tigercosmos** [comment]
thanks.
